packages/react/src/components/collapsible/tests/CollapsibleHeaderIcon.spec.tsx

```diff
@@ -1,69 +0,0 @@
-import {svg} from '@coveord/plasma-style';
```

Comment:
Those tests are useless ?

Reply:
Yes, the component is tested through CollapsibleContainerConnected.spec.tsx, its the only component that uses it, and it is tested with RTL tests.